src/sentry/snuba/spans_metrics.py

```diff
@@ -31,7 +31,7 @@ def query(
     extra_columns=None,
 ):
     builder = SpansMetricsQueryBuilder(
-        dataset=Dataset.Metrics,
+        dataset=Dataset.PerformanceMetrics,
```

Comment:
we want to query the entity prefixed with "generic_" here https://github.com/getsentry/sentry/blob/2981c5968e9894d7778a8f7d1e3608743a447341/src/sentry/search/events/builder/metrics.py#L440